bull-common/src/test/java/com/hotels/transformer/utils/ReflectionUtilsTest.java

```diff
@@ -578,6 +602,29 @@ public void testMapGenericFieldTypeWorksProperly() {
         assertEquals(expectedNestedMapElemType.getGenericClass(), actualNestedMapElemType.getGenericClass());
     }
 
+    @Test
+    public void testMapGenericFieldTypeWorksProperlyForUnparametrizedMap() {
+        // GIVEN
+        MapElemType keyType = ItemType.builder()
+            .objectClass(Map.class)
+            .build();
+        final MapType expectedMapType = new MapType(keyType, keyType);
+        Field field = underTest.getDeclaredField(UNPARAMETRIZED_MAP, FromFooMap.class);
+
+        // WHEN
+        MapType actual = underTest.getMapGenericType(field.getGenericType(), field.getDeclaringClass().getName(), field.getName());
+        ItemType expectedMapKeyType = (ItemType) expectedMapType.getKeyType();
```

Comment:
could you please move the "expected" variables in the "// GIVEN" section?
The same applies to line 618

Reply:
This is done.